sequence_transform.c

```diff
@@ -262,10 +270,118 @@ void st_handle_result(st_trie_t *trie, st_trie_payload_t *res) {
         case 3:  // disable auto-wordbreak
             ends_with_wordbreak = false;
     }
-    if (ends_with_wordbreak) {
+    if (ends_with_wordbreak && st_key_buffer_get_keycode(&key_buffer, 0) != KC_SPC) {
```

Comment:
this needs a comment

Reply:
Indeed. Especially because this is a really hacky way of doing things now. Added.